.github/workflows/pull-request.yml

```
@@ -280,20 +281,25 @@ jobs:
-    needs: [Flutter-Testing]
+    needs: Flutter-Testing
     steps:
       - uses: actions/checkout@v4
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: "3.29.3"
+          flutter-version: ${{ env.FLUTTER_VERSION }}
           channel: "stable" # or: 'beta', 'dev' or 'master'
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)